components/financial_assistance/app/domain/financial_assistance/operations/families/drop_member.rb

```diff
@@ -10,9 +10,9 @@ class DropMember
         send(:include, Dry::Monads[:result, :do])
 
         def call(params:)
-          values = yield validate(params)
+          values = yield validate(params[:applicant_params])
```

Comment:
In any operation we want call method as simple as possible, could pass the params and pick required params inside the method.